app/Models/Server.php

```diff
@@ -130,6 +130,44 @@ public static function createFromPterodactylResponse(Response $response, User $u
         ]);
     }
 
+    /**
+     * Sync the ressources of a specifiy pterodactyl server to CPGG
+     * @param Server $server
+     */
+    public static function syncPterodactylServerSpecs(Server $server): Server
+    {
+        $client = app(PterodactylClient::class);
+        $data = $client->getServer($server->pterodactyl_id)->body();
+        $data = json_decode($data, true);
+        try {
+            $server->pterodactyl_id = $data['attributes']['id'];
+            $server->identifier = $data['attributes']['identifier'];
+            $server->name = $data['attributes']['name'];
+            $server->description = $data['attributes']['description'];
+            $server->status = $data['attributes']['status'];
+            $server->suspended = $data['attributes']['suspended'];
+            $server->memory = $data['attributes']['limits']['memory'];
+            $server->cpu = $data['attributes']['limits']['cpu'];
+            $server->swap = $data['attributes']['limits']['swap'];
+            $server->disk = $data['attributes']['limits']['disk'];
+            $server->io = $data['attributes']['limits']['io'];
+            $server->threads = $data['attributes']['limits']['threads'];
+            $server->oom_disabled = $data['attributes']['limits']['oom_disabled'];
+            $server->databases = $data['attributes']['feature_limits']['databases'];
+            $server->backups = $data['attributes']['feature_limits']['backups'];
+            $server->allocations = $data['attributes']['feature_limits']['allocations'];
+            $server->node_id = $data['attributes']['node'];
+            $server->allocation_id = $data['attributes']['allocation'];
+            $server->nest_id = $data['attributes']['nest'];
+            $server->egg_id = $data['attributes']['egg'];
+
+        } catch (PterodactylRequestException $exception) {
+            //throw exception if it's not a 404 error
+            if ($exception->getCode() !== 404) throw $exception;
```

Comment:
at 404 you should delete the server, not nuke the entire command

Reply:
what if pterodactyl times out during that time? it would delete all following servers